content/projects/kotlin/project-5/data-binding-viewmodel-livedata/_index.md

```diff
@@ -0,0 +1,378 @@
+---
+available_flavours:
+- kotlin
+content_type: project
+prerequisites:
+  hard:
+  - projects/kotlin/project-5/live-data-observers
+  soft: []
+ready: true
+submission_type: repo
+title: Data binding with ViewModel and LiveData.
+---
+
+### App overview
+
+In the this project, you develop the GuessTheWord app, beginning with starter code. GuessTheWord is a two-player charades-style game, where the players collaborate to achieve the highest score possible.
+
+The first player looks at the words in the app and acts each one out in turn, making sure not to show the word to the second player. The second player tries to guess the word.
+
+To play the game, the first player opens the app on the device and sees a word, for example "guitar," as shown in the screenshot below.
+
+
+### 2. App overview
+
+In the this project, you develop the GuessTheWord app, beginning with starter code. GuessTheWord is a two-player charades-style game, where the players collaborate to achieve the highest score possible.
+
+The first player looks at the words in the app and acts each one out in turn, making sure not to show the word to the second player. The second player tries to guess the word.
+
+To play the game, the first player opens the app on the device and sees a word, for example "guitar," as shown in the screenshot below.
+
+![](697975f88441847a.png)
+
+The first player acts out the word, being careful not to actually say the word itself.
+
+- When the second player guesses the word correctly, the first player presses the Got It button, which increases the count by one and shows the next word.
+
+- If the second player can't guess the word, the first player presses the Skip button, which decreases the count by one and skips to the next word.
+
+- To end the game, press the End Game button. (This functionality isn't in the starter code for the first codelab in the series.)
+
+In this codelab, you improve the GuessTheWord app by integrating data binding with LiveData in ViewModel objects. This automates the communication between the views in the layout and the ViewModel objects, and it lets you simplify your code by using LiveData.
+
+
+
+![Title screen](4abf85983013d4f3.png)
+
+![Game screen](4abf85983013d4f3.png)
```

Comment:
game screen and title screen are the same pic?

Reply:
yes